src/efs_parser/BaseFile.py

```diff
@@ -115,13 +123,15 @@ def get_bytes_http(self, offset, size):
         Returns:
             binary string from file
         """
-        if self.local:
+        if self.file_source == "local":
             f = open(self.file, "rb")
             f.seek(offset)
             bin_value = f.read(size)
             f.close()
             return bin_value
-        else:
+        elif self.file_source == "s3":
+            pass
```

Comment:
why is this `pass` ? should this fetch/generate range requests ?

Reply:
The function **get_bytes_http** that contains this line is not used anywhere in the code.
I suggest removing it.

Reply:
Done